nix/packages/ghciwatch.nix

```
@@ -90,6 +103,81 @@
       inherit cargoArtifacts;
     };
 
+  releaseArgs =
+    commonArgs
+    // {
+      # Don't run tests; we'll do that in a separate derivation.
+      # This will allow people to install and depend on `ghciwatch`
+      # without downloading a half dozen different versions of GHC.
+      doCheck = false;
+
+      # Only build `ghciwatch`, not the test macros.
+      cargoBuildCommand = "cargoWithProfile build";
+
+      passthru = {
+        inherit GHC_VERSIONS checks devShell user-manual user-manual-tar-xz;
+      };
+    };
+
+  ghciwatch-with-clap-markdown = craneLib.buildPackage (releaseArgs
+    // {
+      cargoExtraArgs = "--locked --features clap-markdown";
+    });
+
+  cli-markdown = stdenv.mkDerivation {
+    pname = "ghciwatch-cli-markdown";
+    inherit (commonArgs) version;
+
+    phases = ["buildPhase"];
+
+    nativeBuildInputs = [ghciwatch-with-clap-markdown];
+
+    buildPhase = ''
```

Very minor suggestion:

```suggestion
    phases = ["installPhase"];

    nativeBuildInputs = [ghciwatch-with-clap-markdown];

    installPhase = ''
```